sql/core/src/test/scala/org/apache/spark/sql/execution/ui/SQLAppStatusListenerSuite.scala

```diff
@@ -919,7 +922,8 @@ class SQLAppStatusListenerSuite extends SharedSparkSession with JsonTestUtils
 
```

Comment:
Can we leverage `waitUntilEmpty` in `LiveListenerBus`?

Reply:
> Can we leverage `waitUntilEmpty` in `LiveListenerBus`?

Nice suggestion! updated.